Src/ModelLoadCMO.cpp

```diff
@@ -579,6 +584,11 @@ std::unique_ptr<Model> DirectX::Model::CreateFromCMO(
             if (!*nBones)
                 throw std::runtime_error("Animation bone data is missing\n");
 
+            ModelBone::Collection bones;
+            bones.resize(*nBones);
+            auto transforms = ModelBone::MakeArray(*nBones);
+            auto invTransforms = ModelBone::MakeArray(*nBones);
+
             for (UINT j = 0; j < *nBones; ++j)
```

Comment:
Did this not get updated to uint32_t in other cleanups for a reason?

Reply:
I'll do another pass on CMO, VBO, and SDKMESH.